### PR TITLE
fix prices for bitfinex exchanger

### PR DESCRIPTION
**Pull request by alexpalyan**
fix increment value generator for bitfinex + little change for error handler

### CODE REVIEW

**Comment by DeviaVir** [comment]
@nedievas would you mind reviewing?

File: extensions/exchanges/bitfinex/exchange.js [hunk 1]
```diff
@@ -129,6 +129,19 @@ module.exports = function container (get, set, clear) {
       ws_orders['~' + cid].status = 'rejected'
       ws_orders['~' + cid].reject_reason = 'balance'
     }
+    if (error[6] === 'ERROR' && error[7] == 'Invalid price.') {
```
Comment (nedievas) [inline extensions/exchanges/bitfinex/exchange.js]: == or ===

**Review by nedievas — APPROVED** [review]
Nice approach to dynamic price increment

**Review by DeviaVir — APPROVED** [review]
:+1: thanks @nedievas

**Pull request merged**